### PR TITLE
azure

### PR DESCRIPTION
**Pull request by karelbilek**

### CODE REVIEW

File: internal/main.go [hunk 1]
```diff
@@ -270,12 +281,17 @@ func printCoverage(path string, lineOut func(...string)) error {
...
-		return err
+		return 0, err
 	}
 
 	chev := shColor("gray", "❯")
 	cover := sf("%.2f", coverage) + "%"
 
 	lineOut(sf("%s Coverage: %s", chev, shColor(coverageColor(coverage)+":bold", cover)))
-	return nil
+	//err = az.sendCoverage(cover)
```
Comment (alex-parra) [inline internal/main.go]: Comment to remove?

File: internal/azure.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+	CommentType     int    `json:"commentType"`
+}
+
+type AzureConf struct {
+	URL, Auth string
+}
+
+func makeComment(coverage float64, badTests []string) string {
+	coverageComment := "Total coverage is " + sf("%.2f", coverage) + "%"
+	testComment := "All tests are successful. 💪\n\n"
```
Comment (alex-parra) [inline internal/azure.go:28]: 💪🏻

Comment (karelbilek) [reply internal/azure.go:28]: It actually works even :D

File: internal/main.go (hunk 1 above)
Comment (karelbilek) [reply internal/main.go]: Ah yes. Thanks

**Review by alex-parra — APPROVED** [review]
Thanks

**Pull request merged**